smarts/core/sumo_road_network.py

```diff
@@ -387,12 +413,14 @@ def foes(self) -> List[RoadMap.Lane]:
         def waypoint_paths_for_pose(
             self, pose: Pose, lookahead: int, route: RoadMap.Route = None
         ) -> List[List[Waypoint]]:
+            """Waypoints on this lane leading on from the given pose."""
```

Comment:
I suggest using the wording from `road_map.py` here and for `waypoint_paths_at_offset()` below too.  (Also in `opendrive_road_network.py`.)

Reply:
I can just delete this since it inherits the `docstring` from `RoadMap.Lane`.

Reply:
In a way, I am surprised that some methods are not in the interface.